CONTRIBUTING.md

```diff
@@ -83,4 +83,21 @@ This should launch a Koop server on port 8080. You can verify this via the follo
 - http://localhost:8080/marklogic/rest/services/GDeltExample/FeatureServer/0 = should return a JSON feature service 
   descriptor
 
+## Testing the documentation locally
 
+The docs for this project are stored in the `./docs` directory as a set of Markdown files. These are published via
+[GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) using the
+configuration found under "Settings / Pages" in this repository. 
+
+You can build and test the docs locally by 
+[following these GitHub instructions](https://docs.github. 
+com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll), though
+you don't need to perform all of those steps since some of the files generated by doing so are already in the 
+`./docs` directory. You just need to do the following:
+
+1. Install the latest Ruby (rbenv works well for this).
```

Comment:
@BillFarber  If you feel up for it, give this about 10 minutes and see if you can get it working. Took me awhile to get it working, but I'm hoping that now that I have the Jekyll project files included under "docs", then all you need to do is get Ruby 3.2.1 and Jekyll installed, and things will just work.

Reply:
10 minutes ????

1 hour later ...

Reply:
Ok, complete. Everything worked.